src/test/java/com/google/api/codegen/protoannotations/testdata/java_library_no_gapic_config.baseline

```diff
@@ -2238,7 +2765,7 @@ public class LibraryServiceSettings extends ClientSettings<LibraryServiceSetting
     /**
      * Returns the builder for the settings used for calls to listStrings.
      */
-    public UnaryCallSettings.Builder<ListStringsRequest, ListStringsResponse> listStringsSettings() {
+    public PagedCallSettings.Builder<ListStringsRequest, ListStringsResponse, ListStringsPagedResponse> listStringsSettings() {
```

Comment:
These settings getters are also present in java_library.baseline: https://github.com/googleapis/gapic-generator/blob/eea42da690a5cec5660a76c540a45465323128af/src/test/java/com/google/api/codegen/gapic/testdata/java/java_library.baseline#L4571